app/server/lib/FlexServer.ts

```diff
@@ -1286,7 +1286,7 @@ export class FlexServer implements GristServer {
     }
 
     this.app.get('/logout', ...this._logoutMiddleware(), expressWrap(async (req, resp) => {
-      const signedOutUrl = new URL(getOrgUrl(req) + 'signed-out');
+      const signedOutUrl = new URL(getOriginUrl(req) + 'signed-out');
```

Comment:
`getOriginUrl(req)` returns a string that doesn't end with a slash for me, causing the resulting URL to be invalid (`http://localhost:8484signedout`).
Digging a bit to understand if it's true that `getOriginUrl` never ends with a slash, in which case it'll be as easy as changing `signed-out` to `/signed-out`.

Reply:
OK, since `getOriginUrl` is just `` `${protocol}://${host}` `` I think it's pretty safe to add a slash here:
```suggestion
      const signedOutUrl = new URL(getOriginUrl(req) + '/signed-out');
```

Reply:
The `URL` constructor takes [a second argument](https://nodejs.org/docs/latest/api/url.html#new-urlinput-base) to make the first argument relative to.

So it should be `new URL('signed-out', getOriginUrl(req))` and that will handle the problem with trailing slash or not.

Reply:
Duh, of course, thanks @jordigh ! I wonder if it isn't better to still keep the `/` before `sign-out`, to clarify that the expected path should be absolute, since that's the point of this change after all, to make the URL fixed.

```suggestion
      const signedOutUrl = new URL('/signed-out', getOriginUrl(req));
```

Reply:
Oh, yes, missed the context, to make it absolute prefix it with a slash.